keps/sig-apps/2185-random-pod-select-on-replicaset-downscale/README.md

```diff
@@ -263,37 +263,36 @@ _This section must be completed when targeting alpha to a release._
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can a rollout fail? Can it impact already running workloads?**
-  Try to be as paranoid as possible - e.g., what if some components will restart
-   mid-rollout?
+  Rollouts should have no effect, as this only relates to scaledown of replicasets
 
 * **What specific metrics should inform a rollback?**
+N/A
 
 * **Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?**
-  Describe manual testing that was done and the outcomes.
-  Longer term, we may want to require automated upgrade/rollback tests, but we
-  are missing a bunch of machinery and tooling and can't do that now.
+  N/A
 
 * **Is the rollout accompanied by any deprecations and/or removals of features, APIs, 
 fields of API types, flags, etc.?**
-  Even if applying deprecation policies, they may still surprise some users.
+  No
 
 ### Monitoring Requirements
 
 _This section must be completed when targeting beta graduation to a release._
 
 * **How can an operator determine if the feature is in use by workloads?**
-  Ideally, this should be a metric. Operations against the Kubernetes API (e.g.,
-  checking if there are objects with field X set) may be a last resort. Avoid
-  logs or events for this purpose.
+  Only by the presence of the feature gate
 
 * **What are the SLIs (Service Level Indicators) an operator can use to determine 
 the health of the service?**
   - [ ] Metrics
     - Metric name:
     - [Optional] Aggregation method:
     - Components exposing the metric:
-  - [ ] Other (treat as last resort)
-    - Details:
+  - [x] Other (treat as last resort)
+    - Details: The health of this feature is dependent on the overall health of the 
```

Comment:
This isn't helpful - it's true for almost any feature.

What we need is a signal to determine if the kcm is doing what it should be doing.

Reply:
Can we come up with some metrics for this? Even if whitebox-y, that's still much better than nothing.

Maybe we can expose histogram of something like:
- |age of deleted pod| / |age of yougest pod|
[We expect all values to be within [1,2], so having values outside of it is a good signal that something is wrong.

[Well - technically it will collide with https://github.com/kubernetes/enhancements/tree/master/keps/sig-apps/2255-pod-cost so maybe we additionally need a metric label whether such annotation was set or not]
[And then the SLO would be that there are no values >2 when the pod_cost_set=false]

Reply:
> |age of deleted pod| / |age of yougest pod|

are these currently available, or is there work needed to add them to KCM? @soltysh where are the KCM metrics set up so that we can start work on that?